src/storage/slab/slab.c

```diff
@@ -162,6 +162,18 @@ _slab_put_item_into_freeq(struct item *it, uint8_t id)
     PERSLAB_INCR(id, item_free);
 }
 
+/*
+ * Because a slab can have a reserved item (claimed but not linked), which is
```

Comment:
Old comment should be updated to match new purpose of this function.

Reply:
Done.